templates/kubernetes/1/rancher-compose.yml

```diff
@@ -0,0 +1,74 @@
+.catalog:
+  name: "Kubernetes"
+  version: "v1.2.4-rancher4"
+  description: "Rancher Kubernetes service"
+  minimum_rancher_version: v1.1.0
```

Comment:
@alena1108 @ibuildthecloud yes just use the "dev" version in the template, so that it is returned for both dev5 and 1.1